apps/api/src/server.ts

```diff
@@ -37,31 +57,49 @@ export const createServer = () => {
     .use(urlencoded({ extended: true }))
     .use(json())
     .use(cors())
-    .post('/prepare-backup', async (req, res, next) => {
+    .post('/backup-sessions/authenticate', async (req, res, next) => {
       try {
-        const { accessToken }: IAuthResponse = await portablClient.getAccessToken();
+        // const { accessToken }: IAuthResponse = await portablClient.getAccessToken();
 
-        return res.json({ accessToken });
+        const requestUrl = `${PORTABL_API_BASE_URL}/provider/backup-sessions/authenticate`;
+        const requestBody: IAuthenticateBackupSessionRequestDto = {
+          env: PORTABL_ENV,
+          clientId: PORTABL_CLIENT_ID,
+          clientSecret: PORTABL_CLIENT_SECRET,
+        };
+        const { data } = await axios.post<IAuthenticateBackupSessionResponseDto>(requestUrl, requestBody);
+
+        return res.json({ accessToken: data?.accessToken });
       } catch (e) {
+        console.log('111 e', e);
```

Comment:
Let's remove this log, or if you think it's helpful to users setting up quickstart we could potentially log something out here in case of an error.

Reply:
improved